releases/v1.5.0-rc.toml

```diff
@@ -89,5 +89,44 @@ containers the same set of permissions as the calling process, preventing errors
 when privileged containers attempt to get more permissions than given to the
 caller.
 
+#### Configuration changes
+
+##### *New* `registry.config_path` for CRI plugin
+
+`registry.config_path` specifies a directory to look for registry hosts
+configuration. When resolving an image name during pull operations, the CRI
+plugin will look in the `<registry.config_path>/<image hostname>/` directory
+for host configuration. A `hosts.toml` file in that directory may be used
```

Comment:
A `hosts.toml` file -> An optional `hosts.toml` file

Reply:
we should probably surface this too? https://github.com/containerd/containerd/blob/261c107ffc4ff681bc73988f64e3f60c32233b37/remotes/docker/config/hosts.go#L262-L264